src/components/Header.js

```diff
@@ -1,10 +1,29 @@
+import { Link } from "react-router-dom";
+
 const Header = () => {
   return (
     <>
       <div className="header">
-        <h1>John Hutcheon News</h1>
-        <h4>Unbiased reports since 2022</h4>
+        <h1>J H N</h1>
       </div>
+      <div className="sub-header">John Hutcheon News </div>
+      <br></br>
+      <div className="est">est 2022</div>
+
+      <section>
+        <nav>
+          <Link to="/">Home</Link>
+          {" | "}
+          <Link to="/articles"> Articles </Link>
+          {" | "}
+          <Link to="/topics/football"> Football </Link>
+          {" | "}
+          <Link to="/topics/coding"> Coding </Link>
+          {" | "}
+          <Link to="/topics/cooking"> Cooking </Link>
+          {" | "}
+        </nav>
+      </section>
```

Comment:
Do you have a `/api/topics` endpoint?

It would be good to use it to populate this list

Imagine if topics got added/edited/removed as your app grows - you'd have to keep editing your coding and redeploying!
If you use the API it will always be up-to-date